pkgs/applications/audio/tonelib-metal/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, alsa-lib
+, freetype
+, libglvnd
+, mesa
+, curl
+, libXcursor
+, libXinerama
+, libXrandr
+, libXrender
+, libjack2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tonelib-metal";
+  version = "1.1.0";
+
+  src = fetchurl {
+    url = "https://www.tonelib.net/download/220218/ToneLib-Metal-amd64.deb";
```

Comment:
Is there a better URL which contains the version?

Reply:
@SuperSandro2000  yes

Reply:
so? Can we use it?

Reply:
![image](https://user-images.githubusercontent.com/86075850/160285108-b2de28a7-9676-4abf-9e12-f000c45b3e73.png)

Well, the remaining Tonelib builds with the same link, all version programs are different, '220218' shows the release of the release as I understood, but to control the versions of the programs right with the links in any way, just for the release of fresh build, well, on the site Serovno, it is written what version, You can control facing the site. I do not understand what the problem is, all the builds once again repeat with the same reference and this is normal.

Reply:
Well, I'm watching them, all builds with such links, nothing can be done, since I made the manufacturers of software, I look at all buildings on freshness and make PR already on them not the first ten.